Fabulous.XamarinForms/src/Fabulous.XamarinForms.Controls/Xamarin.Forms.Core.json

```diff
@@ -1628,8 +1628,7 @@
           "name": "UseSafeArea",
           "inputType": "bool",
           "modelType": "bool",
-          "defaultValue": "false",
-          "updateCode": "updateUseSafeArea"
+          "updateCode": "(fun _ _ target -> Xamarin.Forms.PlatformConfiguration.iOSSpecific.Page.SetUseSafeArea((target : Xamarin.Forms.Page).On<Xamarin.Forms.PlatformConfiguration.iOS>(), true) |> ignore)"
```

Comment:
This part here.

Reply:
Could you revert this change, please? :)